gallery/src/data-visualization/ReactVisPage.tsx

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { GalleryDescriptionTile } from '../gallery-components/GalleryTiles';
+import { NxTable, NxTableBody, NxTableCell, NxTableHead, NxTableRow,
+  NxWarningAlert } from '@sonatype/react-shared-components';
+
+const ReactVisPage = () =>
+  <>
+    <GalleryDescriptionTile >
+      <p className="nx-p">
+        React-vis is a React-based data visualization library. It is highly customizable, easy-to-use, and very
+        React-friendly. The library is composed of React components that are used to render common data visualization
+        charts like line/area/bar charts, scatterplots, pie and donut charts etc.
+      </p>
+      <h3 className="nx-h2">React-vis Basics</h3>
+      <h3 className="nx-h3">XYPlot</h3>
+      <p className="nx-p">
+        Every series chart that React-vis generates (excludes RadialChart, Sankey, Sunburst and Treemap)
+        is inside a parent component called <code className="nx-code">XYPlot</code>
```

Comment:
Missing space after `XYPlot`.

Rather than add `{' '}` as you have in many other places, it is often easier to choose your line break such that the JSX element is not at the start or end of a line.

Reply:
> Missing space after `XYPlot`.
> 
> Rather than add `{' '}` as you have in many other places, it is often easier to choose your line break such that the JSX element is not at the start or end of a line.

I have tried to space out line breaks as much as possible to avoid using `{' '}`